src/@next/components/containers/Money/Money.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { I18nContext } from "..";
+import { IProps } from "./types";
+
+export const Money: React.StatelessComponent<IProps> = ({
+  money,
+  defaultValue,
```

Comment:
defaultValue is not obligatory in both Money and TaxedMoney.

Maybe we could change type so the look like this:
```
export interface IProps {
  money?: {
    amount: number;
    currency: string;
  } | string;
}
```
And then function signature would look like this

```
({money = "-"}:IProps) => {}
```

We are passing defaultValue as "-" string and if we actually pass some value to money then we will use second type. What do you think?
```